submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
```

Comment:
`attrObj` - what if we use this naming `htmlAttributes` ? Seems like more clear and without useless shortening =)